### PR TITLE
[NEW] Marketplace apps page new list view layout

### PR DESCRIPTION
**Pull request by rique223**
## Proposed changes (including videos or screenshots)

Refactored the layout of the marketplace list of apps, now it has a more minimalist and flexbox-based style. Also implemented a new status filter.

Demo gif:
new-app-list

ClickUp task: 
https://app.clickup.com/t/1na7437

### CODE REVIEW

File: apps/meteor/client/views/admin/apps/AppsFilters.tsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+	const shouldFiltersStack = useMediaQuery('(max-width: 1060px)');
+
+	return (
+		<>
+			<FilterByText placeholder={t('Search_Apps')} onChange={({ text }): void => setText(text)} shouldFiltersStack={shouldFiltersStack}>
+				<RadioDropDown
+					group={freePaidFilterStructure}
+					onSelected={freePaidFilterOnSelected}
+					mie={shouldFiltersStack ? '' : 'x8'}
```
Comment (PedroRorato) [inline apps/meteor/client/views/admin/apps/AppsFilters.tsx:55]: How about creating a variable for this? The same code repeats many times.

File: apps/meteor/client/views/admin/apps/AppsList.tsx [hunk 2]
```diff
@@ -95,57 +80,65 @@ const AppsTable: FC<{
...
+				selectedCategories={selectedCategories}
+				onSelected={onSelected}
+				sortFilterStructure={sortFilterStructure}
+				sortFilterOnSelected={sortFilterOnSelected}
+				categoryTagList={categoryTagList}
+				statusFilterStructure={statusFilterStructure}
+				statusFilterOnSelected={statusFilterOnSelected}
+			/>
+
+			{isAppListReadyOrLoading && (
```
Comment (PedroRorato) [inline apps/meteor/client/views/admin/apps/AppsList.tsx:114]: How about breaking this in different components to reduce the complexity?

File: apps/meteor/client/views/admin/apps/AppsFilters.tsx (hunk 1 above)
Comment (rique223) [reply apps/meteor/client/views/admin/apps/AppsFilters.tsx:55]: Solved

File: apps/meteor/client/views/admin/apps/AppsList.tsx (hunk 2 above)
Comment (rique223) [reply apps/meteor/client/views/admin/apps/AppsList.tsx:114]: Solved

**Review by PedroRorato — APPROVED** [review]

**Review by thassiov — APPROVED** [review]
LGTM

**Pull request merged**